go/obscuronode/enclave/core/crypto.go

```diff
@@ -30,17 +30,12 @@ func EncryptTx(tx *nodecommon.L2Tx) nodecommon.EncryptedTx {
 	return bytes
 }
 
-func encryptTransactions(transactions L2Txs) nodecommon.EncryptedTransactions {
-	result := make([]nodecommon.EncryptedTx, 0)
-	for i := range transactions {
-		result = append(result, EncryptTx(&transactions[i]))
-	}
-	return result
-}
+//func DecryptRollup(rollup *nodecommon.Rollup) *Rollup {
```

Comment:
@tudor-malene Did you mean to leave this in?

Reply:
it was unusued, but I figured it might be needed some day. But it's trivial, so I might as well remove it